skops/io/_persist.py

```diff
@@ -54,7 +54,7 @@ def save(obj, file):
     """
     with tempfile.TemporaryDirectory() as dst:
         with open(Path(dst) / "schema.json", "w") as f:
-            state = get_state(obj, dst)
+            state = _get_state(obj, dst)
```

Comment:
why aren't we using `get_state` (with the new def) here?

Reply:
Yes, that should work, I did a simple search/replace here, I made the change now.